reana-cluster-dev.yaml

```diff
@@ -0,0 +1,73 @@
+cluster:
+  type: "kubernetes"
+  config: "./development-kubeconfig.yaml"
+  config_context: "minikube"
+  version: "v1.6.4"
+  url: "http://localhost"
+
+components:
+  reana-workflow-controller:
+    type: "docker"
+    image: "reanahub/reana-workflow-controller:v0.0.1"
+    mountpoints:
+      - type: hostPath
+        name: devcomponents
+        path: "/devcomponents/reana-workflow-controller:/code/reana-workflow-controller"
```

Comment:
Regarding `devcomponents` naming, locally I have everything under `reana` directory but this name wouldn't work since [we already use this path for simulating the "distributed file system"](https://github.com/reanahub/reana-cluster/blob/master/reana_cluster/backends/kubernetes/templates/deployments/workflow-controller-template.yaml#L25). I would go for `code` as inside the container.

Reply:
Yes, a shared directory named `/code` inside the minikube VM would be nicely consistent with the location of the code inside the containers. However, there is a subtle difference regarding subdirectories that may be a possible source of confusion, in that `/code/reana-job-controller/...` will get mounted under `/code/...` in the job controller container. If we care that this may indeed be confusion-prone(?!), then we could use a neutral name for the VM mount, e.g. `/src`, so that `/src/reana-job-controller` will get mounted under `/code` in the job controller container. I'm not sure I care that much though... both seem fine.

(Disclaimer: I'm using locally `~/<project>/src/...` to store source repositories.)